src/main/java/jenkins/plugins/gerrit/AbstractGerritSCMSource.java

```diff
@@ -511,7 +511,7 @@ private GerritRestApi getGerritClient(GerritURI remoteUri) throws IOException {
               credentials.username,
               credentials.password);
       return new GerritRestApiFactory()
-          .create(authData, SSLNoVerifyCertificateManagerClientBuilderExtension.INSTANCE);
```

Comment:
Can you make it instead just configurable ?
It was added because, unfortunately, some in-house Gerrit setups do not have *official* SSL certs and thus any internal use from Jenkins would fail.

The default should be safe, I agree: SSL validation fully performed and enabled.